app/javascript/components/ansible-credentials-form/ansible-credentials-form.schema.js

```diff
@@ -48,7 +48,8 @@ const createSchema = (fields, promise, edit, loadSchema) => ({
               credential_types: { embedded_ansible_credential_types },
             },
           }) =>
-            Object.keys(embedded_ansible_credential_types).map((key) => ({
+            Object.keys(embedded_ansible_credential_types).filter((key) =>
+              key.includes('ManageIQ::Providers::EmbeddedAnsible::AutomationManager')).map((key) => ({
```

Comment:
I'm not great at JS but it looks like you have a list of `embedded_ansible_credential_types` credential types that you then filter down to only include `EmbeddedAnsible` ones.  Would it make more sense to just update `embedded_ansible_credential_types` to only include EmbeddedAnsible ones from the start?

Where is `embedded_ansible_credential_types` set?

Reply:
It's coming in from API `http://localhost:3000/api/authentications`
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/24a2cf80-a651-44d7-8e1b-cd7adffba85c)

Reply:
Okay this is the issue then https://github.com/ManageIQ/manageiq/blob/master/app/models/authentication.rb#L84-L87
I suggest we fix there rather than here in the UI

Reply:
@DavidResende0 https://github.com/ManageIQ/manageiq/pull/22598

Reply:
@agrare I was using the same endpoint in https://github.com/ManageIQ/manageiq-ui-classic/pull/8834 in order to access the workflow credential types. Should there be a new endpoint created for worflows then or maybe a way to filter when calling `get` in the first place?